### PR TITLE
[v1.28] Add release timeline and initialize team

### PR DESCRIPTION
**Pull request by gracenng**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
Commencing 1.28 documentation including timeline and leads

/assign @kubernetes/sig-release-leads @leonardpahlke

### CODE REVIEW

File: releases/release-1.28/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+title: "Kubernetes 1.28 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.27
```
Comment (leonardpahlke) [inline releases/release-1.28/README.md:11]: v1.28

File: releases/release-1.28/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+| Release Notes complete — reviewed & merged to https://github.com/kubernetes/kubernetes | Release Notes Lead | Tuesday 15th August 2023 | week 14  | |
+| **v1.28.0 released**                                          | Branch Manager | Tuesday 15th August 2023                                                                                               | week 14  | |
+| Release blog published                                        | Comms | Tuesday 15th August 2023                                                                                                        | week 14  | |
+| **[Thaw]**                                                    | Branch Manager | Tuesday 15th August 2023                                                                                               | week 14  | |                                                                                                  | week 15   | |
+
+## Phases
+
+Please refer to the [release phases document](../release_phases.md).
+
+[k8s127-calendar]: https://bit.ly/k8s-release-cal
```
Comment (leonardpahlke) [inline releases/release-1.28/README.md:93]: v1.28

File: releases/release-1.28/exceptions.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+# / Hossein Salahi ([@hosseinsalahi](https://github.com/hosseinsalahi))
+
+# Enhancements Freeze Exceptions requested in 1.27
+enhancementFreeze:
+
+# None!
+
+# Code Freeze Exceptions requested in 1.27
+
+codeFreeze:
```
Comment (leonardpahlke) [inline releases/release-1.28/exceptions.yaml:16]: I think we can leave it empty for now

**Review by leonardpahlke — COMMENTED** [review]
A few things I noticed which we could update 👍

File: releases/release-1.28/exceptions.yaml [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Exception requests in 1.27
```
Comment (furkatgofurov7) [inline releases/release-1.28/exceptions.yaml:1]:
```suggestion
# Exception requests in 1.28
```

File: releases/release-1.28/exceptions.yaml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+# Google Group: https://groups.google.com/g/kubernetes-sig-release
+# Release Team Lead: Xander Grzywinski ([@salaxander](https://github.com/salaxander))
+# Release Team Shadows : Kat Cosgrove ([@katcosgrove](https://github.com/katcosgrove))
+# / Priyanka Saggu ([@Priyankasaggu11929](https://github.com/Priyankasaggu11929))
+# / Heba Elayoty ([@helayoty](https://github.com/helayoty))
+# / Hossein Salahi ([@hosseinsalahi](https://github.com/hosseinsalahi))
+
+# Enhancements Freeze Exceptions requested in 1.27
```
Comment (furkatgofurov7) [inline releases/release-1.28/exceptions.yaml:9]:
```suggestion
# Enhancements Freeze Exceptions requested in 1.28
```

File: releases/release-1.28/exceptions.yaml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+enhancementFreeze:
+
+# None!
+
+# Code Freeze Exceptions requested in 1.27
```
Comment (furkatgofurov7) [inline releases/release-1.28/exceptions.yaml:14]:
```suggestion
# Code Freeze Exceptions requested in 1.28
```

File: releases/release-1.28/release-team.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Kubernetes 1.28 Release Team
+
+| **Team/Role** | **Lead Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
+|----------|----------------------------------|----------------------------------------|
+| Release Team Lead | Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`) | Angelos Kolaitis ([@neoaggelos](https://github.com/neoaggelos)/ Slack: `@Angelos Kolaitis`), Heba Elayoty ([@helayoty](https://github.com/helayoty) / Slack: `@helayoty`), Mark Rossetti ([@marosset](https://github.com/marosset) / Slack: `@Mark Rossetti`), Mickey Boxell ([@mickeyboxell](https://github.com/mickeyboxell)|
```
Comment (furkatgofurov7) [inline releases/release-1.28/release-team.md:5]:
```suggestion
| Release Team Lead | Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`) | Angelos Kolaitis ([@neoaggelos](https://github.com/neoaggelos)/ Slack: `@Angelos Kolaitis`), Heba Elayoty ([@helayoty](https://github.com/helayoty) / Slack: `@helayoty`), Mark Rossetti ([@marosset](https://github.com/marosset) / Slack: `@Mark Rossetti`), Mickey Boxell ([@mickeyboxell](https://github.com/mickeyboxell))|
```

**Review by furkatgofurov7 — COMMENTED** [review]
Few nits, otherwise looks great:

**Review by leonardpahlke — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**